src/app/components/Results/Results.jsx

```diff
@@ -48,17 +48,17 @@ class Results extends React.Component {
 
   getResultsSort() {
     return sortingOpts.map((d, i) => {
-      return (<li role="region" key={i} onClick={e => this.sortResultsBy(e, d.val, d.label)}>
-        {d.label}
+      return (<li role="region" key={i}>
+        <a href="#" onClick={e => this.sortResultsBy(e, d.val, d.label)}>{d.label}</a>
```

Comment:
Also unsure if it's uncool to do an href="#", but I was following @mgiraldo's style guide. Is there something better? For a while, I had plaintext but the a tag was required for nice styling.